prost-build/src/lib.rs

```diff
@@ -352,6 +353,11 @@ impl Config {
         self
     }
 
+    pub fn use_idiomatic_enum_values(&mut self, idiomatic: bool) -> &mut Self {
```

Comment:
Add a method doc about what this does; the prose from the commit message does a good job explaining it.